lib/assets/javascripts/cartodb/models/table.js

```diff
@@ -1164,4 +1140,4 @@
     }
   });
 
-})();
+})();
```

Comment:
Be careful with these missing end of file lines! It's always best to have them :)